lib/full-site-editing/default-template-types.php

```diff
@@ -15,71 +15,71 @@ function gutenberg_get_default_template_types() {
 		),
 		'page'           => array(
 			'title'       => _x( 'Page', 'Template name', 'gutenberg' ),
-			'description' => __( 'Used when an individual Page is queried', 'gutenberg' ),
+			'description' => __( 'Template used to render static pages.', 'gutenberg' ),
 		),
 		'archive'        => array(
 			'title'       => _x( 'Archive', 'Template name', 'gutenberg' ),
-			'description' => __( 'Used when multiple entries are queried. This template will be overridden by the Category, Author, and Date templates where appropriate', 'gutenberg' ),
+			'description' => __( 'The archive template displays multiple entries at once. It is used as a fallback for the Category, Author, and Date templates, which take precedence when they are available.', 'gutenberg' ),
 		),
 		'author'         => array(
-			'title'       => _x( 'Author Archive', 'Template name', 'gutenberg' ),
-			'description' => __( 'Used when a list of Posts from a single author is queried', 'gutenberg' ),
+			'title'       => _x( 'Author', 'Template name', 'gutenberg' ),
+			'description' => __( 'Archive template used to display a list of posts from a single author.', 'gutenberg' ),
 		),
 		'category'       => array(
-			'title'       => _x( 'Post Category Archive', 'Template name', 'gutenberg' ),
-			'description' => __( 'Used when a list of Posts from a category is queried', 'gutenberg' ),
+			'title'       => _x( 'Category', 'Template name', 'gutenberg' ),
+			'description' => __( 'Archive template used to display a list of posts from the same category.', 'gutenberg' ),
 		),
 		'taxonomy'       => array(
-			'title'       => _x( 'Taxonomy Archive', 'Template name', 'gutenberg' ),
-			'description' => __( 'Used when a list of posts from a taxonomy is queried', 'gutenberg' ),
+			'title'       => _x( 'Taxonomy', 'Template name', 'gutenberg' ),
+			'description' => __( 'Archive template used to display a list of posts from the same taxonomy.', 'gutenberg' ),
 		),
 		'date'           => array(
-			'title'       => _x( 'Date Archive', 'Template name', 'gutenberg' ),
-			'description' => __( 'Used when a list of Posts from a certain date are queried', 'gutenberg' ),
+			'title'       => _x( 'Date', 'Template name', 'gutenberg' ),
+			'description' => __( 'Archive template used to display a list of posts from a specific date.', 'gutenberg' ),
 		),
 		'tag'            => array(
-			'title'       => _x( 'Tag Archive', 'Template name', 'gutenberg' ),
-			'description' => __( 'Used when a list of Posts with a certain tag is queried', 'gutenberg' ),
+			'title'       => _x( 'Tag', 'Template name', 'gutenberg' ),
+			'description' => __( 'Archive template used to display a list of posts with a given tag.', 'gutenberg' ),
 		),
 		'attachment'     => array(
 			'title'       => __( 'Media', 'gutenberg' ),
-			'description' => __( 'Used when a Media entry is queried', 'gutenberg' ),
+			'description' => __( 'Used to display individual media items or attachements.', 'gutenberg' ),
```

Comment:
Typo: "attachements".

Also, should it be "Template used to" since most of the other descriptions start that way?